scania-digital-design-system/sdds
# Fix issue rights on Mac for branch #216

**Describe pull-request**  
_Describe what the pull-request is about_
Fix for rights issue on mac, added chmod 777 on husky folder

**Solving issue**  
_Add which issue this pull-request solves by adding # plus the number of the issue (for example #123)_
Helping out on PR #216 for Mac

**How to test**  
_Add description how to test if possible_
1. Try a fresh clone
2. npm i
3. check out this branch
4. Try and do a small change and commit (no push) for testing out husky
5. Should run all lint commands
